emscripten.py

```diff
@@ -1840,7 +1840,7 @@ def create_fp_accessors(metadata):
 
 def create_named_globals(metadata):
   if not shared.Settings.RELOCATABLE:
-    return ''
+    return ''.join(["Module['_%s'] = %s;\n" % (k, v) for k, v in metadata['namedGlobals'].items()])
```

Comment:
It looks like this adds, in non-relocatable code, a bunch of things exported on Module, that can't be DCE'd. Is that right?

Reply:
(what ends up in `namedGlobals` exactly?)

Reply:
Any global addresses that the user asked to export via `EMSCRIPTEN_KEEPALIVE` or `EXPORTED_FUNCTIONS`.

Currently for the wasm backend we add `__data_end` to this list so that binareyen as access to it.  I think I can exclude that here to make this change a NOP in terms of code size for users that don't ask for any exported globals.